tests/ui_tools/test_boxes.py

```diff
@@ -50,6 +50,9 @@ def write_box(
         write_box.model.max_stream_name_length = 60
         write_box.model.max_topic_length = 60
         write_box.model.max_message_length = 10000
+        write_box.model.typing_started_wait_period = 10000
+        write_box.model.typing_stopped_wait_period = 5000
+        write_box.model.typing_started_expiry_period = 15000
```

Comment:
These are the previous constant values? If so, it would be good to add a comment for these three lines, and maybe use the constants themselves as example values?